packages/redux/src/products/selectors/listings.ts

```diff
@@ -760,3 +761,49 @@ export const getProductListingFacetGroups = createSelector(
   state: StoreState,
   hash?: string | number | null,
 ) => FacetGroupDenormalized[] | undefined;
+
+/**
+ * Returns all facets by parent type.
+ *
+ * @param state - Application state.
+ * @param facetGroupType - Facet group type to find.
+ *
+ * @returns Array with all facets content requested by group type.
+ */
+export const getFacetsByGroupType = (
```

Comment:
I think it is better to use a `createSelector` from `reselect` here to avoid returning new instances even when the values haven't changed, which will avoid consumers to find out if there was a change or not between two consecutive calls to this selector. Check other selectors that use `createSelector` to know how to do this.

Reply:
Done!